spec/client_spec.rb

```diff
@@ -1,16 +1,26 @@
 require File.expand_path(File.dirname(__FILE__) + '/spec_helper')
 
 describe TokyoApi::Client do
-
-  describe "instantiated" do
+  describe 'initialization' do
```

Comment:
typo? We're referring to `initialisation_options` in the other file

Reply:
good question! i was thinking of the ruby method `initialize`, but you're probably right that we should stick to the S spellings instead of the Z ones for consistency